NEWS.md

```diff
@@ -19,6 +19,7 @@ New language features
   a `Manifest-v1.11.toml` would be used by v1.11 and `Manifest.toml` by every other julia
   version. This makes managing environments for multiple julia versions at the same time
   easier ([#43845]).
+* `Project.toml` files now support a `[sources]` section for specifying paths and repository locations for dependencies ([#3783](https://github.com/JuliaLang/Pkg.jl/pull/3783))
```

Comment:
I think it is misleading to show only the PR number of the external repository.
Below are some past examples:
https://github.com/JuliaLang/julia/blob/9b20acac2069c8a374c89c89acd15f20d0f2a7ae/NEWS.md?plain=1#L145-L146